src/components/RoomView/roomView.css

```diff
@@ -162,3 +149,19 @@ body {
   justify-content: center;
   margin-top: 2%;
 }
+
```

Comment:
new flexbox organizer of roomview components.

when heidi adds media queries, it'll change the order / size of components